recipe/build.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 set -ex
 
+if [ "${mpi}" == "openmpi" ]; then
+   export OPAL_PREFIX=$PREFIX
+   export OMPI_MCA_plm=isolated
+   export OMPI_MCA_btl_vader_single_copy_mechanism=none
+   export OMPI_MCA_rmaps_base_oversubscribe=yes
+fi
+
```

Comment:
Since we are not testing (running anything after the build), is this really necessary? I am not opposing it, just curious why it had to be added...

Reply:
Same thing here, setting the `OPAL_PREFIX` to the host platform environment for the MPI wrappers for the build platform allows to correctly find and compile., The `OMPI_*` is just baggage from copy&paste, doesn't hurt to have a working OpenMPI environment available here, though.